src/components/SQFormGuidedWorkflow/Types.ts

```diff
@@ -34,7 +34,7 @@ export interface SQFormGuidedWorkflowAdditionalInformationProps
 export interface SQFormGuidedWorkflowAgentScriptProps
   extends SQFormGuidedWorkflowHeaderProps {
   /** Scripted Text for the user to read */
-  text: string;
+  text: React.ReactNode;
```

Comment:
It makes a bit more sense when you look at the structure of the object as a whole. At least to me, anyway

```
const taskModules = [
  {
    ...otherStuff
  scriptedTextProps: {
    title: 'someString',
    text: <>whatever</>
  }
  }
]
```

Reply:
Agreed with both of you. By itself text is very misleading until you obtain the context of the object text lives inside of, which isn't possible to see from the git diff here.

I had to check the `type`, but just in case anyone else may have expected the `type` to be 

`text: React.ReactNode | string`

Here's why the union is not needed:

```typescript
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```